lighthouse-core/gather/gather-runner.js

```diff
@@ -251,8 +260,7 @@ class GatherRunner {
         // We dont need to hold up the reporting for the reload/disconnect,
         // so we will not return a promise in here.
         driver.reloadForCleanStateIfNeeded(options).then(_ => {
-          log.log('status', 'Disconnecting from browser...');
-          driver.disconnect();
+          GatherRunner.disposeDriver(driver);
```

Comment:
Are you sure? I feel it's ok in dispose because of another source runs dispose we still want to show the log disconnecting because it's tightly coupled to disconnect?